manageability-docker/TBH/Dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:20.04 as base
+WORKDIR /src/
+ENV container docker
+RUN if [ ! -z "$http_proxy" ]; then echo "container=docker" >> /etc/environment; fi
+RUN if [ ! -z "$http_proxy" ]; then echo "http_proxy=$http_proxy" >> /etc/environment; fi
+RUN if [ ! -z "$https_proxy" ]; then echo "https_proxy=$https_proxy" >> /etc/environment; fi
+RUN if [ ! -z "$HTTP_PROXY" ]; then echo "HTTP_PROXY=$HTTP_PROXY" >> /etc/environment; fi
+RUN if [ ! -z "$HTTPS_PROXY" ]; then echo "HTTPS_PROXY=$HTTPS_PROXY" >> /etc/environment; fi
+RUN if [ ! -z "$NO_PROXY" ]; then echo "NO_PROXY=$NO_PROXY" >> /etc/environment; fi
+RUN if [ ! -z "$no_proxy" ]; then echo "no_proxy=$no_proxy" >> /etc/environment; fi
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y init mosquitto cryptsetup openssl dmidecode dbus vim net-tools rsyslog libffi-dev pciutils lxc-common dkms libssl-dev libpython3.8 ca-certificates wget openssh-server && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+RUN find /etc/systemd/system \
+    /lib/systemd/system \
+    -path '*.wants/*' \
+    -not -name '*journald*' \
+    -not -name '*systemd-tmpfiles*' \
+    -not -name '*systemd-user-sessions*' \
+    -exec rm \{} \;
+
+RUN systemctl set-default multi-user.target
+
+
+STOPSIGNAL SIGRTMIN+3
+
+COPY ./Intel-Manageability.preview.tar.gz /src/Intel-Manageability.preview.tar.gz
+COPY ./*.deb /src/
+COPY ./adapter.cfg /src/adapter.cfg
+RUN mkdir -p /etc/intel-manageability/public
+RUN mkdir -p /etc/intel-manageability/secret
+RUN touch /etc/intel-manageability/public/tpm_flag
+RUN echo FALSE >> /etc/intel-manageability/public/tpm_flag
+#COPY intel-manageability/ /etc/intel-manageability/
+RUN tar -zxvf *.preview.tar.gz
+RUN dpkg --force-all -i ./mqtt*.deb
+RUN dpkg -i ./inbm-dispatcher*.deb
+RUN dpkg -i ./inbm-diagnostic*.deb
+RUN dpkg -i ./inbm-configuration*.deb
+RUN dpkg -i ./inbm-cloudadapter*.deb
+RUN dpkg -i ./inbm-telemetry*.deb
+RUN dpkg -i ./trtl*.deb
+#RUN sed -i 's/Wants/#Wants/g' /etc/systemd/system/mqtt.service
+#RUN sed -i 's/After/#After/g' /etc/systemd/system/mqtt.service
+RUN mkdir -p /lib/firmware/intel-flashless
```

Comment:
Will we need these commented out lines in the future or can they be removed?

Reply:
I would say remove commented out lines